quiche/src/recovery/cubic.rs

```diff
@@ -425,6 +426,10 @@ fn has_custom_pacing() -> bool {
     false
 }
 
+fn state_str(r: &Recovery, now: Instant) -> &'static str {
+    reno::state_str(r, now)
```

Comment:
bit yuck to refer to reno here :D

maybe we could move states that are shared across CC's into a common type?